salt/client/spm.py

```diff
@@ -21,22 +22,23 @@
 import salt.utils
 import salt.utils.http as http
 import salt.syspaths as syspaths
+from salt.utils import parsers
+from salt.ext.six import string_types
+from salt.ext.six.moves import input
+from salt.ext.six.moves import zip
 
 # Get logging started
 log = logging.getLogger(__name__)
 
 
-class SPMClient(object):
+class SPMClient(parsers.SPMParser):
```

Comment:
The only class sub-classing `parsers.SPMParser` should be the one which has direct access to the CLI, ie, `salt.cli.spm`, any specific reason why you're sub-classing it here?